invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -1,402 +1,714 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "id": "http://localhost/schemas/records/record-v1.0.0.json",
-  "title": "Invenio Datacite based Record Schema v1.0.0",
+  "id": "https://inveniosoftware.org/schemas/rdm/records/record-v1.0.0.json",
+  "title": "InvenioRDM Record Schema v1.0.0",
   "type": "object",
   "additionalProperties": false,
 
-  "properties": {
-    "$schema": {
-      "description": "This record's jsonschema.",
-      "type": "string"
+  "definitions": {
+
+    "lang": {
+      "description": "ISO 639-3 language code.",
+      "type": "string",
+      "maxLength": 3
     },
-    "conceptid": {
-      "description": "Invenio record identifier (alphanumeric).",
+
+    "nameType": {
+      "description": "Type of name.",
+      "type": "string",
+      "enum": [
+          "organizational",
+          "personal"
+      ]
+    },
+
+    "titleType": {
+      "description": "Type of title.",
+      "type": "string",
+      "enum": [
+          "alternative_title",
+          "subtitle",
+          "translated_title",
+          "other"
+      ]
+    },
+
+    "contributorType": {
       "type": "string"
     },
-    "id": {
-      "description": "Invenio record identifier (alphanumeric).",
+
+    "dateType": {
+      "description": "Type of the date.",
+      "type": "string",
+      "enum": [
+        "accepted",
+        "available",
+        "copyrighted",
+        "collected",
+        "created",
+        "issued",
+        "submitted",
+        "updated",
+        "valid",
+        "withdrawn",
+        "other"
+      ]
+    },
+
+    "relationType": {
       "type": "string"
     },
-    "access": {
-      "access_right": {
-        "default": "open",
-        "description": "Access right for record.",
-        "type": "string"
-      },
-      "created_by": {
-        "description": "ID of user that created the deposit.",
-        "type": "integer"
-      },
-      "metadata_restricted": {
-        "default": false,
-        "description": "Record metadata accesibility. Public by default (False).",
-        "type": "boolean"
-      },
-      "files_restricted": {
-        "default": false,
-        "description": "Record associated files accesibility. Public by default (False).",
-        "type": "boolean"
-      },
-      "owners": {
-        "description": "List of user IDs that are owners of the record.",
-        "type": "array",
-        "minItems": 1,
-        "uniqueItems": true,
-        "items": {
-          "type": "number"
+
+    "descriptionType": {
+      "type": "string",
+      "enum": [
```

Comment:
How could they be changed? e.g. if someone wants to change any vocab that it's on an enum needs to redefine their jsonschema? Otherwise it won't pass validation, no?

Reply:
To be decided